src/pipeline_configuration_spec.py

```diff
@@ -40,11 +40,12 @@ class RapidProTarget:
 @dataclass
 class PipelineConfiguration:
     pipeline_name: str
+    test_participant_uids: []
```

Comment:
```suggestion
    test_participant_uuids: []
```
for consistency with the db

Reply:
updated thanks.